src/api/zones.py

```
@@ -0,0 +1,267 @@
+                )
+            
+            zone_request = ZoneRequest.from_dict(payload)
+            is_valid, error_msg = zone_request.validate()
+            if not is_valid:
+                return current_app.response_class(
+                    response=json.dumps(ErrorResponse(error_msg).to_dict()),
+                    status=400,
+                    mimetype='application/json'
+                )
+            
+            success, message = zone_service.update_zone_data(
```

Ruff (0.14.4)

Unpacked variable `success` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)